stargazer/APIReference/constellationRPCAPI/wallet_watchAsset.md

```diff
@@ -0,0 +1,66 @@
+---
+hide_table_of_contents: true
+---
+
+<head>
+  <meta
+    name="description"
+    content="Adds an L0 token to the Stargazer wallet."
+  />
+</head>
+
+<intro-end />
+
+Adds an L0 token to the Stargazer wallet.
```

Comment:
Do we support ETH/EVM networks with this RPC method also?

Reply:
This RPC method was only added to the `constellation` provider at the moment. We'll add it to the EVM provider in the future. cc @joaoCarvalho1000